src/renderer/components/Map.tsx

```diff
@@ -62,17 +64,22 @@ function makeStationMarker(
   const diameter = circleDiameter(species.size(), maxSpecies);
```

Comment:
The marker size used to vary depending on the number of observations. If we are not longer doing that, we won't need `diameter` anymore.

We'll also need to change the [anchor point](https://docs.mapbox.com/mapbox-gl-js/api/markers/#marker-parameters) of the markers (it seems to be center by default):
```ts
new mapboxgl.Marker(markerElement, { anchor: 'bottom' })
```

Reply:
Diameter is being used in ".setPopup(new mapboxgl.Popup({ offset: diameter / 2 }).setDOMContent(popupContentPlaceholder));"

Reply:
"diameter" is being used somewhere else in the file, line 128.

Reply:
We use it to offset the popup position, which made sense when the marker had variable size. We should remove it now.

Reply:
Alright. I will remove it.